File: rafael/operadores/UseOperadoresLogicos.java
class UseOperadoresLogicos {
	public static void main(String[] args) {	
		int q = 10;
		int w = 5;
		int e = 8;
		int r = 11;

		System.out.println(q > w | e < r);
		System.out.println(q > r || e < w);
		System.out.println(q > e & w < r);
		System.out.println(q > w && r < e);
		System.out.println(q > w ^ e < r);
	}
}

File: rafael/operadores/UseOperadoresLogicos.cs
class UseOperadoresLogicos 
{
	static void Main() 
	{		
		int q = 10;
		int w = 5;
		int e = 8;
		int r = 11;

		System.Console.WriteLine(q > w | e < r);
		System.Console.WriteLine(q > r || e < w);
		System.Console.WriteLine(q > e & w < r);
		System.Console.WriteLine(q > w && r < e);
		System.Console.WriteLine(q > w ^ e < r);
	}
}
